README.md

```diff
@@ -41,7 +41,7 @@ const ctx = await simulator.request('/test-url', {
 
 ### API
 
-#### `getSimulator(app: FusionApp, testPlugin?: FusionPlugin) => { request, render }`
+#### `getSimulator(app: FusionApp, testPlugin?: FusionPlugin) => { request, render, getService }`
```

Comment:
Thoughts on a similar approach for `getMiddleware` to better support mocking `ctx` and testing in isolation.